README.demo.md

```diff
@@ -35,6 +35,15 @@ If you would like to change the assets, please check the [Laravel Mix documentat
 
 ### Commercial Mesh Plugins
 
+#### Required for some demos
```

Comment:
Please move this section before "### Running the portal" and add a link to https://getmesh.io/docs/plugin-system/#_deployment

Reply:
I have moved this to the "Running the portal" part